.github/workflows/ci.yml

```diff
@@ -86,6 +86,16 @@ jobs:
     - name: Lint Go
       run: make lint-go-full
 
+  vulncheck:
```

Comment:
@aruneshpa,

I thought you were going to run this via the Makefile target? Why not follow the convention used for the other checks? I'd prefer we rewrite this using the following patch:

---

⚠️ **Please note** that I renamed the Makefile target to `vulncheck-go` per another comment I left on this review. The other comment has the reasons for this change.

---

```diff
diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index 86cd9346..07546d2d 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -86,6 +86,28 @@ jobs:
     - name: Lint Go
       run: make lint-go-full
 
+  verify-vulnerabilities:
+    needs:
+    - verify-go-modules
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+    - name: Install Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ env.GO_VERSION }}
+        cache: true
+        cache-dependency-path: '**/go.sum'
+    - name: Setup the cache for golangci-lint
+      uses: actions/cache@v3
+      with:
+        key: verify-vulnerabilities-${{ runner.os }}-go${{ env.GO_VERSION }}-${{ hashFiles('go.sum', 'hack/tools/go.sum') }}
+        path: |
+          hack/tools/bin/vulncheck
+    - name: Vulncheck Go
+      run: make vulncheck-go
+
   build-manager:
     needs:
     - verify-go-modules
```

Reply:
Fixed.